ledfx/config.py

```
@@ -765,6 +765,46 @@ def sanitise_effect_config(effect_type, old_config):
+            effect = virtual.get("effect", {})
+            if effect.get("type") == "equalizer2d":
+                effect["config"] = invert_equalizer2d_flip_vertical(effect.get("config", {}))
+            
+            # Handle stored effects in the effects dictionary
+            effects_dict = virtual.get("effects", {})
+            if "equalizer2d" in effects_dict:
+                effects_dict["equalizer2d"]["config"] = invert_equalizer2d_flip_vertical(effects_dict["equalizer2d"].get("config", {}))
+        
+        # Update user presets for equalizer2d
+        equalizer2d_presets = new_config.get("user_presets", {}).get("equalizer2d", {})
+        for preset_name, preset_data in equalizer2d_presets.items():
```

Ruff (0.13.1)

Loop control variable `preset_name` not used within loop body

Rename unused `preset_name` to `_preset_name`

(B007)